doc/content/source/problems/OpenMCCellAverageProblem.md

```diff
@@ -391,7 +391,76 @@ element in the `[Mesh]`.
 
 While this class mainly facilitates data transfers to and from OpenMC, a number of
 other features are implemented in order to enable more convenient input file
-setup and achieve better iterative performance. These are described in this section.
+setup, achieve better iterative performance, and iteratively update the geometry
+(for CAD tracking only). These are described in this section.
+
+#### CAD Geometry Skinning
+  id=skinning
+
+For all cell-based OpenMC models, OpenMC is currently limited to settings a single
+constant temperature, and a single constant density, in each cell. For CSG geometries,
+the user needs to manually set up sub-divisions in the geometry in order to capture
+spatial variation in temperature and density (computed by some other MOOSE application).
+For DAGMC geometries, you can instead optionally re-generate the OpenMC cells after
+each Picard iteration according to contours in temperature and/or density - in other
+words, automatically setting up the OpenMC model so that it can receive spatially-varying
+temperature and density without the user needing to manually subdivide regions of space
+*a priori*.
+
+!alert note
+This skinning feature is only available for DAGMC geometries for which the `[Mesh]`
+obeys the same material boundaries in the starting `.h5m` file.
+
+To use this feature, provide the `skinner` user
+object parameter, of type [MoabSkinner](/userobjects/MoabSkinner.md). After each OpenMC
+run, this object will group elements in the `[Mesh]` into "bins" for temperature, density,
+and subdomain (mesh block). A new DAGMC cell will then be created by "skinning" the mesh
```

Comment:
We can do all or some of these?

```suggestion
and/or subdomain (mesh block). A new DAGMC cell will then be created by "skinning" the mesh
```

Reply:
In the long run, we'll be able to do all of these simultaneously. I didn't implement the necessary material-reinits for density binning, so `OpenMCCellAverageProblem` currently throws an error if trying to use density binning (just to protect any user who picks this up right away)